x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/risk_score_mapping/default_risk_score.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import { EuiFormRow, EuiText, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiRange } from '@elastic/eui';
+import type { EuiRangeProps } from '@elastic/eui';
+import { MIN_RISK_SCORE, MAX_RISK_SCORE } from './constants';
+import * as i18n from './translations';
+
+interface DefaultRiskScoreProps {
+  value: number;
+  onChange: (newValue: number) => void;
+  errorMessage?: string;
+  idAria?: string;
+  dataTestSubj?: string;
+}
+
+export function DefaultRiskScore({
+  value,
+  onChange,
+  errorMessage,
+  idAria,
+  dataTestSubj = 'defaultRiskScore',
+}: DefaultRiskScoreProps) {
+  const handleChange = useCallback<NonNullable<EuiRangeProps['onChange']>>(
+    (event) => {
+      const newValue = (event.target as HTMLInputElement).value;
```

Comment:
It turns out I can't use `event.currentTarget` because it causes our tests to fail, even though everything still works in the UI. Therefore, I had to revert to the `event.target` approach.